### PR TITLE
More Infinite Supply Vendors

### PR DESCRIPTION
**Pull request by whatston3**
## About the PR

Makes more vendors infinite supply (LessLethalVend, LibertyVend, CiviMed, SalvageVend, most of the AstroVend).  Removes CiviMed resupply, removes vendor resupplies for infinite vendors from dungeons.

~~Changes the value of welding fuel, plasma and uranium in the FuelVend to 750, 3000, and 3500 (the latter two _must_ be more expensive than the lowest price available in the cargo market).~~  Moved elsewhere.

The CiviMed becomes, like the FuelVend, a snarky jerk with high prices.  The goal here is to establish a reasonable baseline for medical services while allowing medical supply to exist when doctors are not around - it will be expensive, and it will suck, but it's there.  The CiviMed receives a wallmount version, and is mapped to replace the existing NanoMed and NanoMed Plus vendors on POIs and stations.  All shuttle vendors have been left alone, though a restock was removed from the Bison (as the salvage vendor itself is now infinite).

I'm putting this up as a proposal, I think the CiviMed prices may need review, and certain items in each inventory may need review, but the concept itself should be feature complete and warrants some discussion, just trying to get the ball rolling.

Full disclosure, I've also pork barreled a small syndicate grammar fix in here.

## Why / Balance

Low pop hours need some source of medical supply, and without restocks, it's difficult to get without buying a ship (which we should not want) or going on expedition.

## How to test

1. Buy things from the CiviMed, peruse the prices.  The sprites should make sense compared to the wallmount NanoMed.
2. Compare the contents of the NanoMed vs. the CiviMed, check the contents of the Salvage vendor, AstroVend, YouTool, EngiVend, Vendomat, LibertyVend, and LessLethalVend.  They should be mostly infinite on POIs.

## Media

Comparison of the NanoMed vs. the CiviMed inventories: note that the CiviMed's stock is infinite.  The NanoMed as-is is being largely repurposed for doctor supplies on medical ships.
image

- [X] ***I have added screenshots/videos to this PR showcasing its changes ingame***, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

:cl:
- tweak: Most POI vendors (save drobes) have an infinite stock of items.

### CODE REVIEW

**Comment by whatston3** [comment]
Drafting this to fix up my crimes.

File: Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/dungeon_items_general.yml [hunk 1]
```diff
@@ -1070,18 +1069,18 @@
...
-    - VendingMachineRestockSalvageEquipment
     - VendingMachineRestockSeeds
     - VendingMachineRestockSmokes
-    - VendingMachineRestockTankDispenser
-    - VendingMachineRestockVendomat
     - VendingMachineRestockRobotics
     - VendingMachineRestockChemVend
     - VendingMachineRestockAstroVend
+<<<<<<< HEAD
+    - VendingMachineRestockFlatpackVend
```
Comment (Cheackraze) [inline Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/dungeon_items_general.yml:1079]: Merge artifact

**Comment by Cheackraze** [comment]
I really don't like seeing vending price used like that because it's going to prevent future market changes. Need to not use that and use the MarketModifier.

The proper way to do it, so that we can allow the same goods to display different prices inside different vendors (nanometer vs civmed) would be to give the civmed a much higher MarketModifier, so it can use identical items but sell at higher than a medical ship/station vending machine intended for medical players.

**Comment by whatston3** [comment]
> I really don't like seeing vending price used like that because it's going to prevent future market changes. Need to not use that and use the MarketModifier.
> 
> The proper way to do it, so that we can allow the same goods to display different prices inside different vendors (nanometer vs civmed) would be to give the civmed a much higher MarketModifier, so it can use identical items but sell at higher than a medical ship/station vending machine intended for medical players.

Fair, a base StaticPrice value should be specified.  The MarketRate of the CiviMed is very high as-is.

Cleaning this up at the moment, pardon the dust.  Also Darin brought up a good point, restricting this to POI versions of the vendors.  Should be in a much better state in pretty short order.

**Comment by Cheackraze** [comment]
> > I really don't like seeing vending price used like that because it's going to prevent future market changes. Need to not use that and use the MarketModifier.
> > The proper way to do it, so that we can allow the same goods to display different prices inside different vendors (nanometer vs civmed) would be to give the civmed a much higher MarketModifier, so it can use identical items but sell at higher than a medical ship/station vending machine intended for medical players.
> 
> Fair, a base StaticPrice value should be specified. The MarketRate of the CiviMed is very high as-is.
> 
> Cleaning this up at the moment, pardon the dust. Also Darin brought up a good point, restricting this to POI versions of the vendors. Should be in a much better state in pretty short order.

cool, this is in preperation for a system that is intended to act on the market modifiers of machines on a grid-grid basis to help simulate our economy. Market goes up - cost of goods goes up but so does payout of selling. And vise versa. Things that bypass that system, such as VendingPrice, has the potential to make sort of exploiting scenarios where the market might move out of range of the VendingPrice value, and run the risk of just buying infinite items and then turning around and selling them straight to the now inflated market, especially when we start introducing 'infinite' vending machines

**Comment by whatston3** [comment]
Removed the FuelVend stuff for now (bigger can of worms, will put it in a separate PR).  I've left most of the vendor.  Linter seems fine on my side, I've left most of the vending crates, but flatpack vendors and the civimed have been removed entirely (all versions are infinite).  I think this is good to go.

**Review by ErhardSteinhauer — APPROVED** [review]
Looks good, works good.

**Pull request merged**